src/igvfd/schemas/changelogs/workflow.md

```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 5
 
+* Add `analysis_step_version`.
```

Comment:
I think there is still a typo here, the property added is `analysis_step_versions`.